R/slide.R

```diff
@@ -19,24 +20,28 @@
 #' @param ... Additional arguments to pass to the function or formula specified
 #'   via `f`. Alternatively, if `f` is missing, then the current argument is
 #'   interpreted as an expression for tidy evaluation. See details.  
-#' @param n Number of time steps to use in the running window. For example, if
-#'   `n = 7`, one time step is one day, and the alignment is "right", then to
-#'   produce a value on January 7 we apply the given function or formula to data
-#'   in between January 1 and 7.
+#' @param before A nonnegative integer specifying the number of time steps
+#'   before each of the `ref_time_values` to extract data from.
+#'   This must be a vector of length 1.
+#'   Set to 0 for a right-aligned/trailing sliding window, meaning
+#'   that no
+#'   `time_value` after the slide will be used for the sliding calculation.
+#'   It is mandatory to specify a `before` value, unless `after` is specified
+#'   as a non-zero value. In this case, `before` will be assumed to be 0, as it
+#'   assumes the user wants to do a left-aligned/leading sliding window.
+#'   However, this usage is discouraged and will thus produce a warning.
+#' @param after A nonnegative integer specifying the number of time steps after
+#'   each of the `ref_time_values` to extract data from.
```

Comment:
Same as previous comment.

And in at least one of these we should give an example with concrete dates the way there was previously.